systemd/nslogin/Makefile

```diff
@@ -2,7 +2,7 @@ SRC=nslogin.c
 LIBS=-lsystemd
 CFLAGS=-Wall -Wextra -Wconversion -pedantic
 CMODEFLAGS=-O0 -g -DDEBUG
-POSTBUILD=sudo chown root:root $@ && sudo chmod a+s $@
+POSTBUILD=sudo chown root:root $@ && sudo chmod ug+s $@
```

Comment:
Makefile should never run `sudo` but assume they are running as root (it will block package build). It can error out, but please keep in mind that you are expected to be able to run under `fakeroot`

Reply:
Got it. Remove sudo is easy.